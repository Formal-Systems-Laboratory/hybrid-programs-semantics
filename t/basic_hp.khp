vars a ; a := a + 1
