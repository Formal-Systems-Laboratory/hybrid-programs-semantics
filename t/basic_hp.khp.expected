<generatedTop>
  <k>
    .
  </k>
  <state>
    a |-> ?I +Int 1
  </state>
</generatedTop>
